test/e2e/tests/editor/editor.pictures.spec.ts

```diff
@@ -51,8 +51,6 @@ test.describe('Editor pictures', () => {
   });
 
   test('Showing and hiding captions', async ({ managerTab, browserName }) => {
-    test.slow(browserName === 'firefox');
```

Comment:
Why did you remove this? We really don't want to have to debug this again, because it sometimes takes too long on Firefox. I'd say just leave it unless you're confident it's really not necessary.

Reply:
Restored in commit dba024ceb. I removed it because it didn't seem necessary in my test runs on my local machine, but it's true that GitHub Actions runners are often slower, so I put it back.